OpenRA.Mods.Common/Commands/DebugVisualizationCommands.cs

```diff
@@ -45,11 +45,11 @@ public void WorldLoaded(World w, WorldRenderer wr)
 				help.RegisterHelp(name, helpText);
 			};
 
-			register("showcombatgeometry", "toggles combat geometry overlay.");
-			register("showrendergeometry", "toggles render geometry overlay.");
-			register("showscreenmap", "toggles screen map overlay.");
-			register("showdepthbuffer", "toggles depth buffer overlay.");
-			register("showactortags", "toggles actor tags overlay.");
+			register("combat-geometry", "toggles combat geometry overlay.");
+			register("render-geometry", "toggles render geometry overlay.");
+			register("screen-map", "toggles screen map overlay.");
+			register("depth-buffer", "toggles depth buffer overlay.");
+			register("actor-tags", "toggles actor tags overlay.");
```

Comment:
It looks to me like you didn't adjust the switch-case in `InvokeCommand` for those?